docs/development.md

```diff
@@ -72,9 +72,22 @@ from the repository root. (This may take a while the first time you run it.)
 
 ## Updating vendored dependencies
 
+The `dep` tool is currently only marginally supported by k/ repos. There are
+some warts.
+
 1.  Use the `dep` tool and/or modify the `Gopkg.toml` file to reference the
-    dependency versions that you need. Refer to the [dep
+    new dependency versions. Refer to the [dep
     docs](https://github.com/golang/dep#usage) for more info.
-1.  Run `dep ensure`.
+1.  Run `dep prune`.
+1.  At this point, there may be other modifications necessary either to the
+    vendored dependencies or the `Gopkg.toml` file. The known ones are noted
+    below. Make these and any additional necessary ones, and add them to this
+    list.
+    -   As of #58, it was necessary to modify the BUILD file in
+        `vendor/k8s.io/client-go/util/cert` to have the go_library not reference
+        testdata.
 1.  Run the commands from the [Prerequisites](#Prerequisites).
 1.  Run the tests and fix any breakages.
+1.  When sending out a PR, please put the handmade changes in one commit and the
+    mechanical updates in another commit so that it's easier for reviewers to
```

Comment:
I think it will be more straightforward to say generated updates instead of mechanical updates.

Reply:
Done.